File: src/test/java/ting/leetcode/java/hashtable/TopKFrequentElementsTest.java
package ting.leetcode.java.hashtable;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertArrayEquals;

class TopKFrequentElementsTest {
    private TopKFrequentElements topKFrequentElements;

    @BeforeEach
    void setUp() {
        topKFrequentElements = new TopKFrequentElements();
    }

    @Test
    void topKFrequent() {
        assertArrayEquals(topKFrequentElements.topKFrequent(new int[]{ 4, 4, 4, 5, 5, 6 }, 2), new int[]{ 4, 5 });
        assertArrayEquals(topKFrequentElements.topKFrequent(new int[]{ 4, 5 }, 2), new int[]{ 4, 5 });
        System.out.println(Math.round(11.56));
        System.out.println(Math.round(-11.56));
        System.out.println(Math.round((-4 + 0.4D)));
    }
}

File: src/test/kotlin/ting/leetcode/hashtable/TopKFrequentElementsTest.kt
package ting.leetcode.hashtable

import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions.*

internal class TopKFrequentElementsTest {
    private lateinit var topKFrequentElements: TopKFrequentElements

    @BeforeEach
    fun setUp() {
        topKFrequentElements = TopKFrequentElements()
    }

    @Test
    fun topKFrequent() {
        assertArrayEquals(topKFrequentElements.topKFrequent(intArrayOf(4, 4, 4, 5, 5, 6), 2), intArrayOf(4, 5))
        assertArrayEquals(topKFrequentElements.topKFrequent(intArrayOf(4, 5), 2), intArrayOf(4, 5))
    }
}